jfix-stdlib-ratelimiter/src/main/java/ru/fix/stdlib/ratelimiter/ConfigurableRateLimiter.java

```diff
@@ -0,0 +1,64 @@
+package ru.fix.stdlib.ratelimiter;
+
+import io.github.resilience4j.ratelimiter.RateLimiterConfig;
+import io.github.resilience4j.ratelimiter.internal.AtomicRateLimiter;
+
+import java.time.Duration;
+import java.time.temporal.TemporalUnit;
+
+
+public class ConfigurableRateLimiter implements RateLimiter {
+
+    private volatile io.github.resilience4j.ratelimiter.RateLimiter rateLimiter;
+
+    public ConfigurableRateLimiter(String name, int permitsPerSecond) {
+        // AtomicRateLimiter does not distribute events inside refresh period.
+        // It releases all available permits immediately on interval start if there is a demand for them.
+        // To distribute event in our interval of 1 second we divide it
+        // into chunks so that 1 chunk of time is limited to 1 permit.
+        rateLimiter = new AtomicRateLimiter(
+            name,
+            RateLimiterConfig.custom()
+                .limitForPeriod(1)
+                .limitRefreshPeriod(Duration.ofNanos(1_000_000_000 / permitsPerSecond))
+                .build()
+        );
+    }
+
+    @Override
+    public boolean tryAcquire() {
+        return rateLimiter.getPermission(Duration.ZERO);
+    }
+
+    @Override
+    public boolean tryAcquire(long timeout, TemporalUnit unit) {
+        return rateLimiter.getPermission(Duration.of(timeout, unit));
+    }
```

Comment:
Thread could enter interrupted state during acquire.  
Maybe we should rise InterruptedException in such case? 
```
val result = rateLimiter.getPermission(Duration.of(timeout, unit));
if (Thread.interrupted()) {
      throw new InterruptedException()
}
return result;
```